ops/model.py

```diff
@@ -2630,8 +2761,10 @@ def send_signal(self, sig: Union[int, str], *service_names: str):
             service_names: Name(s) of the service(s) to send the signal to.
 
         Raises:
-            pebble.APIError: If any of the services are not in the plan or are
-                not currently running.
+            APIError: if any of the services are not in the plan or are not currently running.
+            ConnectionError: if pebble cannot be reached.
+            APIError: if an error occurred communicating with pebble.
+            TypeError: if no service names are provided.
```

Comment:
We'll be dropping these per above, but also let's get rid of the `TypeError` as discussed. That's just a programmer error and we don't want folks catching it.